iset.mm

```diff
@@ -143029,6 +143048,53 @@ could not be used in an extensible structure (slots must be positive
       /\ ( TopSet ` ndx ) e. NN ) $=
     ( cts c9 df-tset 9nn ndxslid ) ABCDE $.
 
+  $( The index of the slot for the group operation in an extensible structure
+     is a positive integer.  (Contributed by AV, 31-Oct-2024.) $)
+  tsetndxnn $p |- ( TopSet ` ndx ) e. NN $=
+    ( cnx cts cfv c9 cn tsetndx 9nn eqeltri ) ABCDEFGH $.
+
+  $( The index of the slot for the base set is less then the index of the slot
+     for the topology in an extensible structure.  (Contributed by AV,
+     31-Oct-2024.) $)
+  basendxlttsetndx $p |- ( Base ` ndx ) < ( TopSet ` ndx ) $=
+    ( c1 c9 cnx cbs cfv cts clt 1lt9 basendx tsetndx 3brtr4i ) ABCDECFEGHIJK $.
+
+  $( The slot for the topology is not the slot for the base set in an
+     extensible structure.  (Contributed by AV, 21-Oct-2024.)  (Proof shortened
+     by AV, 31-Oct-2024.) $)
+  tsetndxnbasendx $p |- ( TopSet ` ndx ) =/= ( Base ` ndx ) $=
+    ( cnx cbs cfv cts basendxnn nnrei basendxlttsetndx gtneii ) ABCZADCIEFGH $.
+
+  $( The slot for the topology is not the slot for the group operation in an
+     extensible structure.  (Contributed by AV, 18-Oct-2024.) $)
+  tsetndxnplusgndx $p |- ( TopSet ` ndx ) =/= ( +g ` ndx ) $=
+    ( cnx cts cfv cplusg wne c9 2re 2lt9 gtneii tsetndx plusgndx neeq12i mpbir
+    c2 ) ABCZADCZEFNENFGHIOFPNJKLM $.
+
+  $( The slot for the topology is not the slot for the ring multiplication
+     operation in an extensible structure.  (Contributed by AV,
+     31-Oct-2024.) $)
+  tsetndxnmulrndx $p |- ( TopSet ` ndx ) =/= ( .r ` ndx ) $=
+    ( cnx cts cfv cmulr wne c9 c3 3re 3lt9 gtneii tsetndx mulrndx neeq12i mpbir
+    ) ABCZADCZEFGEGFHIJOFPGKLMN $.
+
+  $( The slot for the topology is not the slot for the involution in an
+
+     extensible structure.  (Contributed by AV, 11-Nov-2024.) $)
```

Comment:
empty line?

Reply:
Fixed. Looks like something went wrong in my edit to the comment or the rewrapper or the combination.